src/app/routes/bridge.ts

```diff
@@ -13,19 +14,41 @@ class BridgeController {
   }
 
   async getUsage(req: Request, res: Response) {
-    const usage = await this.service.User.getUsage((req as AuthorizedRequest).user);
+    let user = (req as AuthorizedRequest).user;
+    const { workspaceUserId } = req.query;
```

Comment:
Do not destructure if this param is optional, the code could crash on execution time. When getting information from an endpoint do this instead:

`const workspaceUserId = req.query.workspaceUserId`